libs/web/shared/ui/src/lib/box/box.tsx

```diff
@@ -52,21 +83,17 @@ export const Box = component$<BoxProps>((props) => {
     '24': 'gap-96',
   } satisfies { [K in BoxGap]: string };
 
-  const Borders = {
-    none: 'rounded-none',
-    full: 'rounded-full',
-    '1': 'rounded',
-    '2': 'rounded-md',
-    '3': 'rounded-lg',
-    '4': 'rounded-xl',
-    '5': 'rounded-2xl',
-    '6': 'rounded-3xl',
-  } satisfies { [K in BoxBorder]: string };
+  const Shadows = {
+    none: 'shadow-none',
+    sm: 'shadow-sm',
+    md: 'shadow-md',
+    lg: 'shadow-lg',
+  } satisfies { [K in BoxShadow]: string };
 
   const additionalProps = new PropsBuilder(props).withSize().withPadding().build();
 
   return (
-    <div {...rest} class={['flex', Directions[direction], Variants[variant], Alignments[align], Gaps[gap], Borders[border], additionalProps]}>
+    <div {...rest} class={[gridDirection ? 'grid' : 'flex', Directions[direction], Variants[variant], Alignments[align], Gaps[gap], gridDirection === 'row' ? GridRows[grid] : GridCols[grid], Shadows[shadow], additionalProps]}>
```

Comment:
I guess we can improve this line better than ternary, for more understandable and clean

Reply:
What do you mean by **better than ternary**?

Reply:
Like the way we done at `PropsBuilder` it could be check at other function rather than in `class` and push to `results` array then add it to class later. For make the `class` property clean and single-responsibility for that

Reply:
I guess you can use the check at `PropsBuilder` and pass the args to that to return checked params.